x-pack/filebeat/input/gcs/input_test.go

```diff
@@ -662,7 +663,7 @@ func Test_StorageClient(t *testing.T) {
 					if !tt.checkJSON {
 						val, err = got.Fields.GetValue("message")
 						assert.NoError(t, err)
-						assert.True(t, tt.expected[val.(string)])
+						assert.True(t, tt.expected[strings.ReplaceAll(val.(string), "\r", "")])
```

Comment:
why do we need this change in this PR?

Reply:
it's because windows treating line breaks as `\r\n` instead of `\n` which results in tests failing on windows